testdata/p4_16_samples_outputs/psa-dpdk-checksum-arg-header.p4.spec

```diff
@@ -0,0 +1,187 @@
+
+
+struct ethernet_t {
+	bit<48> dst_addr
+	bit<48> src_addr
+	bit<16> ether_type
+}
+
+struct ipv4_t {
+	bit<8> ver_ihl
+	bit<8> diffserv
+	bit<16> total_len
+	bit<16> identification
+	bit<16> flags_offset
+	bit<8> ttl
+	bit<8> protocol
+	bit<16> hdr_checksum
+	bit<32> src_addr
+	bit<32> dst_addr
+}
+
+struct vxlan_t {
+	bit<8> flags
+	bit<24> reserved
+	bit<24> vni
+	bit<8> reserved2
+}
+
+struct udp_t {
+	bit<16> src_port
+	bit<16> dst_port
+	bit<16> length
+	bit<16> checksum
+}
+
+struct dpdk_pseudo_header_t {
+	bit<4> pseudo
```

Comment:
non-byte-aligned struct field

Reply:
it's coming from pass which try to lower the Checksum

Reply:
fixed